tensorflow-base-cpu-linwin/meta.yaml

```diff
@@ -123,7 +119,6 @@ requirements:
     - numpy >=1.20  # [not aarch64]
     - numpy >=1.19  # [aarch64]
     - opt_einsum 3.3.0.*
-    - protobuf >=3.9.2
```

Comment:
About protobuf https://github.com/tensorflow/tensorflow/blob/2ea19cbb575d076b4f521d3603211c8316ad5f8f/tensorflow/tools/pip_package/setup.py#L84-L91

Reply:
Hm, that's too bad. We need to build against protobuf 3.20, so I cannot put that constraint there. If there is a problem on Windows, we'll have a way to fix it.